shared/logutil/logutil.go

```diff
@@ -0,0 +1,83 @@
+package logutil
+
+import (
+	"os"
+	"strings"
+
+	joonix "github.com/joonix/log"
+	"github.com/sirupsen/logrus"
+	prefixed "github.com/x-cray/logrus-prefixed-formatter"
+)
+
```

Comment:
It's usually a good idea to show that the WriterHook implements the Hook interface. 

Add this line
```suggestion

var _ = logrus.Hook(&WriterHook{})

```

If the WriterHook doesn't conform to that interface, then the package will fail to build.

Reply:
Got it.